source/plugins/ruby/in_kube_podinventory.rb

```diff
@@ -258,13 +286,27 @@ def parse_and_emit_records(podInventory, serviceRecords, continuationToken, batc
               containerInventoryRecords.each do |cirecord|
                 if !cirecord.nil?
                   containerInventoryStream.add(emitTime, cirecord) if cirecord
+                  ciRecordSize = cirecord.to_s.length
+                  @winContainerInventoryTotalSizeBytes += ciRecordSize
+                  if ciRecordSize >= 65536 # 64KB
+                    @winContainerCountWithInventoryRecordSize64KBOrMore += 1
+                  end
+                  if !cirecord["EnvironmentVar"].nil? && !cirecord["EnvironmentVar"].empty? && cirecord["EnvironmentVar"].length >= 65536
+                    @winContainerCountWithEnvVarSize64KBOrMore += 1
+                  end
+                  if !cirecord["Ports"].nil? && !cirecord["Ports"].empty? && cirecord["Ports"].length >= 65536
+                    @winContainerCountWithPortsSize64KBOrMore += 1
+                  end
+                  if !cirecord["Command"].nil? && !cirecord["Command"].empty? && cirecord["Command"].length >= 65536
```

Comment:
Maybe the 65536 should be changed to be (64 * 1024) or made a constant just to be more clear about where the number is coming from

Reply:
> Maybe the 65536 should be changed to be (64 * 1024) or made a constant just to be more clear about where the number is coming from

> fixed pr feedback.